### PR TITLE
Update Profile Page with new labels

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description
As someone using the Comparison Tool, I want to be able to filter search results on more specific criteria so that I can more quickly find the institution I am looking for.

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/11324
## Testing done
local testing

## Screenshots
Screen Shot 2020-08-03 at 1 46 20 PM
## Acceptance criteria
- [x] Gender: Women Only, Men Only
- [x] Specialized Mission: HBCU
- [x] Religious Affiliation: See supporting artifact 1
a. For values of "-1 Not reported" and "-2 Not applicable" no tags will be displayed
b. For values of "99 Other (none of the above)", display the tag "Other religious affiliation"
c. For values of 77 & 79 (Protestant, Not specified) and (Other Protestant), display the tag "Protestant"
- [x] Tags are displayed in the following order:
a. Gender
b. Religious Affiliation
c. HBCU
- [x] The following colors are utilized for the tags:
a. Text: 323A45
b. Background: D6D7D9

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/gi/sass/partials/_gi-profile-page.scss [hunk 1]
```diff
@@ -3,6 +3,15 @@
...
     margin-top: 0;
   }
+  .info-flag {
+    color: #323a45;
+    background-color: #d6d7d9;
+    font-size: 0.9em;
+    display: inline-block;
+    padding: 0.1em 0.5em;
+    margin-right: 0.5em;
+    //margin-top: 1em;
```
Comment (shawkey-daniel) [inline src/applications/gi/sass/partials/_gi-profile-page.scss:13]: Please remove commented code

File: src/applications/gi/sass/partials/_gi-profile-page.scss [hunk 2]
```diff
@@ -359,13 +368,13 @@
...
+    &.usa-accordion-button[aria-expanded="false"] {
+      background-image: url("/img/arrow-up.png");
+      background-image: url("/img/arrow-up.svg");
     }
-    &.usa-accordion-button[aria-expanded='true'] {
-      background-image: url('/img/arrow-down.png');
-      background-image: url('/img/arrow-down.svg');
+    &.usa-accordion-button[aria-expanded="true"] {
+      background-image: url("/img/arrow-down.png");
+      background-image: url("/img/arrow-down.svg");
```
Comment (shawkey-daniel) [inline src/applications/gi/sass/partials/_gi-profile-page.scss:377]: I think single quotes are preferred. Are your lint settings wrong?

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

File: src/applications/gi/sass/partials/_gi-profile-page.scss (hunk 2 above)
Comment (ksantiagomorgan6) [reply src/applications/gi/sass/partials/_gi-profile-page.scss:377]: They might be I'll revert that

File: src/applications/gi/sass/partials/_gi-profile-page.scss (hunk 1 above)
Comment (ksantiagomorgan6) [reply src/applications/gi/sass/partials/_gi-profile-page.scss]: removed

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

**Pull request merged**